1.3.0.CR2 Not finding jar from other subproject

I am trying to upgrade to `1.3.0.CR2` from `1.2.0.Final`, and getting an error where the quarkus project can't find a jar generated from one of the sibling projects.

The project can be found here: https://github.com/GregJohnStewart/task-timekeeper

On an interesting note, `:WebServer:Server:test` works, but `:WebServer:Server:quarkusDev` fails with the following information:

```
Caused by: org.gradle.api.GradleException: Failed to run
        at io.quarkus.gradle.tasks.QuarkusDev.startDev(QuarkusDev.java:399)
        at org.gradle.internal.reflect.JavaMethod.invoke(JavaMethod.java:104)
   ...
Caused by: java.nio.file.FileSystemNotFoundException: /home/gstewart/GitRepos/task-timekeeper/BaseCode/ManagerIO/build/libs/ManagerIO-1.0.0.409.jar
        at com.sun.nio.zipfs.ZipFileSystem.<init>(ZipFileSystem.java:120)
        at com.sun.nio.zipfs.ZipFileSystemProvider.newFileSystem(ZipFileSystemProvider.java:139)
        at io.quarkus.gradle.AppModelGradleResolver.collectDependencies(AppModelGradleResolver.java:256)
        at io.quarkus.gradle.AppModelGradleResolver.resolveModel(AppModelGradleResolver.java:143)
        at io.quarkus.gradle.tasks.QuarkusDev.startDev(QuarkusDev.java:242)
        ... 92 more

```

--- a/devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java
+++ b/devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusDev.java
@@ -62,6 +62,7 @@ import io.quarkus.bootstrap.resolver.AppModelResolverException;
 import io.quarkus.dev.DevModeContext;
 import io.quarkus.dev.DevModeMain;
 import io.quarkus.gradle.QuarkusPluginExtension;
+import io.quarkus.runtime.LaunchMode;
 import io.quarkus.utilities.JavaBinFinder;
 
 public class QuarkusDev extends QuarkusTask {
@@ -235,7 +236,7 @@ public class QuarkusDev extends QuarkusTask {
             StringBuilder classPathManifest = new StringBuilder();
 
             final AppModel appModel;
-            final AppModelResolver modelResolver = extension().getAppModelResolver();
+            final AppModelResolver modelResolver = extension().getAppModelResolver(LaunchMode.DEVELOPMENT);
             try {
                 final AppArtifact appArtifact = extension.getAppArtifact();
                 appArtifact.setPath(extension.outputDirectory().toPath());

